core/Objects/CmfPackage.cs

```diff
@@ -271,6 +271,18 @@ public class CmfPackage : IEquatable<CmfPackage>
         [JsonConverter(typeof(ListAbstractionsDirectoryConverter))]
         public List<IDirectoryInfo> BuildablePackages { get; set; }
 
+        /// <summary>
+        /// Gets or sets the target directory where the dependencies contents should be extracted.
+        /// This is used when the package dependencies are restored in the restore and build commands.
+        /// </summary>
+        /// <value>
+        /// The dependencies target directory.
+        /// </value>
+        [JsonProperty(Order = 23)]
+#nullable enable

```

Comment:
We should either decide to use nullable style or not, mixing it is not great for readability

Reply:
reverted